tests/test_plugin_management/test_parse_plugin_changes.py

```diff
@@ -139,20 +155,18 @@ def test_get_testing_info_testing_not_needed():
 
 
 def test_run_changed_plugin_tests(mocker):
-    changed_files = " ".join(DUMMY_FILES_CHANGED)
-
     plugin_info_dict_mock = mocker.patch("brainscore_core.plugin_management.parse_plugin_changes.parse_plugin_changes")
-    plugin_info_dict_mock.return_value = {'modifies_plugins': True, 'changed_plugins': {'models': ['dummy_model'], 'benchmarks': ['dummy_benchmark'], 'data': [], 'metrics': []}, 'is_automergeable': False, 'run_score': 'True'}
+    plugin_info_dict_mock.return_value = {'modifies_plugins': True, 'test_all_plugins': ['data', 'benchmarks'], 'changed_plugins': {'models': ['dummy_model'], 'benchmarks': [], 'data': [], 'metrics': []}, 'is_automergeable': False, 'run_score': 'True'}
```

Comment:
before this test was only for one benchmark, now it tests data + benchmark -- separate into two tests?